gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,627 @@
+
+[[no-token]]
+** `no valid registration token was found`
++
+To avoid this issue, generate a new token for the command by refreshing the *Add a Flex Gateway* page for Kubernetes and rerun the <<reg-command, registration command>>.
+
+
+[[helm-troubleshooting]]
+=== Helm Command Issues
+
+Troubleshoot errors returned by a helm command:
+
+[[cluster_unreachable]]
+* If your Kubernetes cluster is not running when you attempt to deploy Flex Gateway using `helm`, the following error can occur:
++
+----
+Error: Kubernetes cluster unreachable: 
+the server could not find the requested resource
+----
++
+Start the service that runs your cluster. 
++
+You can check that your Kubernetes cluster host is running your cluster, for example:
++
+[source,kubectl]
+----
+kubectl cluster-info
+----
++
+If the cluster is running, the command returns a result similar to this one:
++
+----
+Kubernetes control plane is running at https://34.30.50.119
+GLBCDefaultBackend is running at https://34.30.50.119/api/v1/namespaces/kube-system/services/default-http-backend:http/proxy
+KubeDNS is running at https://34.30.50.119/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Metrics-server is running at https://34.30.50.119/api/v1/namespaces/kube-system/services/https:metrics-server:/proxy
+----
++
+If the cluster is not up, the command can return a result similar to this one:
++
+----
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+error: the server doesn't have a resource type "services"
+----
++
+For additional information, see <<check-cluster>>.
+
+Troubleshoot errors returned by a `helm` command:
+
+[[upgrade-failed]]
+* If the Helm upgrade command fails to generate a `Service` resource of the `LoadBalancer` type, the command hangs for 5 minutes (due to the `--wait`) flag in the command and produces the following timeout error:
++
+----
+Error: UPGRADE FAILED: timed out waiting for the condition
+----
++
+This error typically means that your cluster configuration is incorrect or that your cluster configuration tool does not support this process. See <<k8-cluster-reqs>>.
+
+* If the `registration.yaml` file is invalid when you pass it to the Helm during deployment of the Flex Gateway container, the following error results:
++
+----
+Error: UPGRADE FAILED: execution error at (flex-gateway/templates/deployment.yaml:4:10):
+registerSecretName, registration.content or registration.secretName is required!
+----
++
+When this error occurs, check the `registration.yaml`. Make sure the file is not empty. If the file is empty, re-register your Flex Gateway container with a new, unique name to generate a new `registration.yaml` (see <<register-flex>>) and rerun the procedures that follow the registration step. 
+
+//TODO: this was from Israel. Not clear what upgrade error and 
```

Comment:
@seblaz Do you have any suggestions for this section? I'm not sure what to do with it.

Reply:
I might put those commands in another doc if they aren't needed in this one.